bayesian/utility/v2/component_analyses.py

```diff
@@ -22,7 +22,7 @@
 import time
 from collections import namedtuple
 from typing import Dict, Set, List, Tuple
-from f8a_utils.dependency_finder import DependencyFinder
+from f8a_utils.tree_generator import GolangDependencyTreeGenerator
```

Comment:
are there no references to the other tree generators like NPM, Maven etc?

Reply:
No. I referenced it myself for Version cleaning Explicitly.
Added in this PR https://github.com/fabric8-analytics/fabric8-analytics-server/pull/682